src/main/java/uk/gov/hmcts/cmc/claimstore/rules/MoreTimeRequestRule.java

```diff
@@ -18,8 +20,14 @@ public void assertMoreTimeCanBeRequested(Claim claim) {
             throw new MoreTimeAlreadyRequestedException("You have already requested more time");
         }
 
-        if (LocalDate.now().isAfter(claim.getResponseDeadline())) {
+        assertIsNotPastDeadline(LocalDateTimeFactory.nowInLocalZone(), claim.getResponseDeadline());
+    }
+
+    void assertIsNotPastDeadline(LocalDateTime now, LocalDate responseDeadline) {
+        LocalDateTime responseDeadlineTime = responseDeadline.atTime(16, 0);
+        if (now.isEqual(responseDeadlineTime) || now.isAfter(responseDeadlineTime)) {
```

Comment:
isn't isAfter enough?

Reply:
If its 4:00 PM it means it's past deadline according to what Marc F. told me. This is aligned with the frontend `dateTime.isSameOrAfter(fourPM(responseDeadline))`

Reply:
ah could have sworn he said 4:01 but 4 should be safe enough